zeebe/broker/src/main/java/io/camunda/zeebe/broker/system/partitions/impl/steps/ExporterDirectorPartitionTransitionStep.java

```diff
@@ -129,7 +129,7 @@ private ActorFuture<Void> openExporter(
     final ExporterDirector director =
         exporterDirectorBuilder.apply(exporterCtx, context.getExporterPhase());
 
-    context.getComponentHealthMonitor().registerComponent(director.getName(), director);
+    context.getComponentHealthMonitor().registerComponent(director);
```

Comment:
`LogStream` is not `HealthMonitorable`, so I guess this was not doing anything.

Reply:
I think you commented on the wrong line but not registering logstream is correct. I refactored the logstream a while ago to not be `HealthMonitorable` but forgot to remove the registration.